### PR TITLE
added log-transform argument to normalize function

### PR DESCRIPTION
**Pull request by LisaSikkema**
good to make explicit that log-transformation is performed. Set default to True, for compatibility with rest of scripts.

### CODE REVIEW

**Review by danielStrobl — APPROVED** [review]
Looks good!

File: scIB/preprocessing.py [hunk 1]
```diff
@@ -173,7 +173,11 @@ def normalize(adata, min_mean = 0.1):
     # modify adata
     adata.obs['size_factors'] = size_factors
     adata.X /= adata.obs['size_factors'].values[:,None]
-    sc.pp.log1p(adata)
+    if logtransform == True:
```
Comment (danielStrobl) [inline scIB/preprocessing.py:176]: I think you could just do `if logtransform:` here

File: scIB/preprocessing.py [hunk 2]
```diff
@@ -143,7 +143,7 @@ def plot_count_filter(adata, obs_col='n_counts', bins=60, lower=0, upper=np.inf,
         plt.show()
 
 ### Normalisation
-def normalize(adata, min_mean = 0.1):
+def normalize(adata, min_mean = 0.1, logtransform=True):
```
Comment (LuckyMD) [inline scIB/preprocessing.py:146]: could we turn this argument into the simpler `log=True`?

**Review by LuckyMD — CHANGES_REQUESTED** [review]
a little simplification of the code needed :)

**Review by LuckyMD — APPROVED** [review]

**Comment by LuckyMD** [comment]
Thanks a lot @LisaSikkema !

**Pull request closed without merging**